src/main/java/org/broadinstitute/hellbender/tools/spark/sv/discovery/BreakpointComplications.java

```diff
@@ -470,9 +633,18 @@ protected BreakpointComplications(final Kryo kryo, final Input input) {
             dupSeqRepeatUnitRefSpan = null;
             dupSeqRepeatNumOnRef = 0;
             dupSeqRepeatNumOnCtg = 0;
+            dupSeqStrandOnRef = null;
+            dupSeqStrandOnCtg = null;
             cigarStringsForDupSeqOnCtg = null;
             dupAnnotIsFromOptimization = false;
         }
+
+        if (input.readBoolean()) {
```

Comment:
Is there a reason not to use `readObjectOrNull` and `writeObjectOrNull` in this serialization code?

Reply:
Nope, except personal preference for native types. Plus `SimpleInternal` is marked Java serializable but not using Kryo.